.dir-locals.el

```diff
@@ -92,4 +92,4 @@
 
  ("bin"
   (clojure-mode
-   (cider-clojure-cli-aliases . "dev"))))
+   (cider-clojure-cli-aliases . "dev:drivers:build:build-dev"))))
```

Comment:
Not strictly part of this fix, but when working on the scripts we want to have these aliases included by default